python-sdk/noxfile.py

```diff
@@ -68,7 +69,7 @@ def test_examples_by_dependency(session: nox.Session, extras):
     session.install("-e", ".[tests]")
     session.run("airflow", "db", "init")
 
-    session.run("pytest", "tests/test_example_dags.py", *pytest_args, *session.posargs)
+    session.run("pytest", "tests/test_example_dags.py", *pytest_args, *session.posargs, external=True)
```

Comment:
I added `external=True` because nox was complaining (with a warning) that this is not in the (I think) required dependencies. This fix simply suppresses the warning as suggested by the warning.

Reply:
Can we add this as a comment here, it will be useful.

Reply:
```suggestion
    # Since pytest is not installed in the nox session directly, we need to set `external=true`.
    session.run("pytest", "tests/test_example_dags.py", *pytest_args, *session.posargs, external=True)
```